### DISCUSSION TITLE
Unable to download YouTube videos

### DISCUSSION THREAD

**Discussion by ytehmj** [aclap-dev/video-downloadhelper#1444] [Help and Q&A]
### Your question

I can't download any youtube videos
like: https://www.youtube.com/watch?v=nIElvf-i2FQ

I need to use a proxy to access youtube, and I don't know if that affects the download

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: zh-CN
coapp: {"found":true,"path":"D:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"b973072c255a4ae5"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:132.0) Gecko/20100101 Firefox/132.0
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "inbrowser",
    "used_history_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": true,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 4,
    "successfull_dl": 32,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_2146836460817725"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Administrator\\dwhelper\\BG East Wrestling-04.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "BG East Wrestling-04.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\Administrator\\dwhelper"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "http://www.bgeast.com/vidondemand-play.php?vid=BBY3-1&ft=yes"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1730993990545
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}

**Comment by Mattmjs** [comment]
It doesn't work with a proxy, try a vpn instead.
